generate-release/src/release_notes.rs

```diff
@@ -107,3 +147,68 @@ file_name = "{file_name}.md"
         title = title.trim().replace('"', "\\\"")
     )
 }
+
+#[allow(clippy::too_many_arguments)]
+fn generate_and_open_issue(
+    client: &GithubClient,
+    issue_titles: &HashSet<String>,
+    pr: &GithubIssuesResponse,
+    title: &str,
+    authors: &[String],
+    file_path: &Path,
+    milestone: &str,
+    dry_run: bool,
+) {
+    let pr_number = pr.number;
+    let issue_title = format!("Write release notes for PR #{pr_number}: {title}");
+
+    // Check if this issue already exists
+    // If it does, we don't want to spam the repo with duplicate issues
+    if issue_titles.contains(&issue_title) {
+        return;
+    }
+
+    let pr_url = format!("https://github.com/bevyengine/bevy/pull/{pr_number}",);
+    let file_path = file_path.to_string_lossy();
+
+    let authors = authors
+        .iter()
+        .map(|author| format!("{}", author))
+        .collect::<Vec<_>>()
+        .join(", ");
```

Comment:
```suggestion
    let authors = authors.join(", ");
```

Clippy doesn't like this line, so here's a simplified version.